### PR TITLE
Improve risk display in reset and error cases (EXPOSUREAPP-6356)

### PR DESCRIPTION
**Pull request by d4rken**
* https://github.com/corona-warn-app/cwa-app-android/commit/b290ddac793c1f78067848929beb4492ca25d0b6 fixes the initial calculation failed state on app reset by changing the "fallback value if there is no data" to LOW_RISK instead of CALCULATION_FAILED, which became an issue after changing CALCULATION_FAILED to be the dominant state.
* https://github.com/corona-warn-app/cwa-app-android/pull/2810/commits/02cbccae93ce6aacb072586960a0c33273778276 which changes the TracingRepo to call both tracing tasks when there is an error. Otherwise pressing the retry button wouldn't help if it was the presence tracing that failed last.

For testing you'd want to reset the app via in app reset, and play a bit with failed risk states (i.e. no internets).

### CODE REVIEW

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/storage/TracingRepository.kt [hunk 1]
```diff
@@ -91,29 +93,17 @@ class TracingRepository @Inject constructor(
...
-                )
+    fun refreshRiskResult() = scope.launch {
+        Timber.tag(TAG).d("refreshRiskResults()")
+
+        riskWorkScheduler.runRiskTasksNow()
+
+        taskController.submit(
+            DefaultTaskRequest(
+                RiskLevelTask::class,
+                originTag = "TracingRepository.refreshDiagnosisKeys()"
```
Comment (chiljamgossow) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/storage/TracingRepository.kt:104]: Should the tag be updated as well?

**Review by chiljamgossow — DISMISSED** [review]
LGTM

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/storage/TracingRepository.kt (hunk 1 above)
Comment (d4rken) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/storage/TracingRepository.kt]: Yes, currently only affects logging though.

Comment (d4rken) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/storage/TracingRepository.kt]: Done.

**Review by chiljamgossow — APPROVED** [review]

**Review by chris-cwa — APPROVED** [review]
LGTM!

**Review by BMItr — APPROVED** [review]
code lgtm 👌 
nothing to complain

**Pull request merged**